### DISCUSSION TITLE
stats command with warning strconv.ParseFloat: parsing "[1107]"

### DISCUSSION THREAD

**Discussion by swartz-k** [juicedata/juicefs#3055] [General]
I use juicefs command version `juicefs version 1.0.2+2022-10-13.514ef03`.

While use `juicefs stats /juicefs --verbosity 1` there is warning log.
```
2022/12/05 09:47:14.059952 juicefs[716] <WARNING>: parse [1021]: strconv.ParseFloat: parsing "[1021]": invalid syntax [stats.go:361]
```

I think this because of `/juicefs/.stats` contains multi type record, as below.
```
meta.reconnects: 24
meta.usec_ping: [1038]
```
Failed logic here.
https://github.com/juicedata/juicefs/blob/fd42b7c0237253c4d134760af558aad6291cd7b9/cmd/stats.go#L359

**Comment by davies** [comment]
The mount point is using hosted JuiceFS, the stats command of open source does not support that.

For the upcoming release of hosted JuiceFS, it will have `stats` command.
